app/metrics.yaml

```
@@ -2653,6 +2653,45 @@ history:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-11-01"
+  search_term_group_open_tab:
+    type: event
+    description: |
+      A user opens a tab from the search term group.
```

From reading these descriptions, I thought these were probes for the tabs tray itself. Might be good to clarify the location where these probes are being added. For example..

```suggestion
      A user opens a tab from the search term group from the history list.
```

---

Agreed.  Looks like these are for History.  FYI, I recently add one telemetry for this requirement https://github.com/mozilla-mobile/fenix/pull/22300